packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -246,6 +319,21 @@ export const addAssetToVault = async (
   // eslint-disable-next-line no-restricted-syntax -- allow this computed property
   await consumeInstance[oracleInstanceName];
 
+  const auctioneerCreator = E.get(auctioneerKit).creatorFacet;
+  const schedules = await E(auctioneerCreator).getSchedule();
+
+  const finishPromiseKit = makePromiseKit();
+  const addBrandThenResolve = ToFarFunction('addBrandThenResolve', async () => {
+    await E(auctioneerCreator).addBrand(interchainIssuer, keyword);
+    finishPromiseKit.resolve(true);
```

Comment:
I don't see this value read. I think it's meant to be:
```suggestion
    finishPromiseKit.resolve(undefined);
```

Reply:
correct